code/modules/detectivework/scanner.dm

```diff
@@ -80,7 +83,10 @@
 
 		scanning = TRUE
 
-		user.visible_message(span_notice("\The [user] points the [src.name] at \the [A] and performs a forensic scan."))
+		user.visible_message(
+			span_notice("\The [user] points the [src.name] at \the [A] and performs a forensic scan."),
+			ignored_mobs = user
+		)
 		to_chat(user, span_notice("You scan \the [A]. The scanner is now analysing the results..."))
```

Comment:
```suggestion
		user.visible_message(
			span_notice("\The [user] points the [src.name] at \the [A] and performs a forensic scan."), \
			span_notice("You scan \the [A]. The scanner is now analysing the results...")
		)
```
There's already functionality built into visible_message for this, is there a reason not to use it?

Reply:
What you have scanned (or forgot to scan) is one of the few things that I scroll up on the chat tab to see so I wanted to keep it universal.

Reply:
Fair enough. Would it then not make sense to also send clearing the log to chat so you know what you've scanned since you last cleared it? Or even better just give the damn thing a way to list out what you scanned?